curated_tokenizers/tests/test_bbpe_processor.py

```diff
@@ -114,3 +115,10 @@ def test_can_load_from_file_object(vocab_path, merges_path):
                 merges=merges,
             )
     assert toy_processor.encode(EXAMPLE_TEXT) == (EXAMPLE_PIECE_IDS, EXAMPLE_PIECES)
+
+
+def test_pickle(toy_processor):
+    serialized = dumps(toy_processor)
+    deserialized = loads(serialized)
```

Comment:
Perhaps we can also double check the correct class type of `deserialized` with an `assert isinstance`?
(same comment for the other 2 tests)